ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -242,7 +241,8 @@ objects:
           wait_for_machineconfigpool_to_apply
 
           setup_ssh_bastion
-
+          # Set up kube ssh bastion to bastion host that was set up recently
+          export KUBE_SSH_BASTION=${BASTION_HOST}
```

Comment:
This var is not used in DR scenarios, why set it here?

Reply:
It's not for DR testing, I am planning to use it in an e2e test - https://github.com/openshift/origin/commit/fe7bff04966a8eb9404a604eb453b3d1998930af#diff-cd46155e5d0497e7b5eda36cd96e26faR3322

Reply:
Right, in this case it should not be set in DR functions anyway

Reply:
So, I don't understand where exactly we setting those variables in those DR functions, are you telling if that variable is set some correct value(which is resolvable and works), those functions will start using `bastion` to run `oc` commands or am I missing something here?

Reply:
You are setting this in the `restore-cluster-state` function which is used for disaster recovery. The equivalent function for general e2e testing is `run-tests` (line 523), although I'm not certain if you should set an envvar inside that function. `restore-cluster-state` doesn't use the bastion or a proxy to execute `oc` commands. It uses the `bastion_ssh` function to execute commands directly on the masters for making/copying etcd snapshots.

Reply:
To be clear, my understanding is that this is an environment variable and it doesn't matter where we're setting. It'll be a problem if we're deleting the value related to it, in this case, we're explicitly removing the bastion host which will cause the problem.